test/input/beamville/beam.conf

```diff
@@ -94,8 +104,8 @@ beam.replanning{
 ##################################################################
 # Warm Mode
 ##################################################################
-beam.warmStart.enabled = false
-beam.warmStart.path = "https://s3.us-east-2.amazonaws.com/beam-outputs/run149-base__2018-06-27_20-28-26_2a2e2bd3.zip"
+beam.warmStart.enabled = true
+beam.warmStart.path = "/Users/haitam/workspace/beam/output/beamville/beamville__2019-05-13_17-18-48/ITERS/it.0"
```

Comment:
Please, make sure this path exist everywhere (maybe need to add it under sourcecontrol)

Reply:
ok